packages/root/src/cli/start.ts

```diff
@@ -69,7 +69,7 @@ export async function createProdServer(options: {
   server.use(sessionMiddleware());
 
   const plugins = rootConfig.plugins || [];
-  configureServerPlugins(
+  await configureServerPlugins(
```

Comment:
can you add the same to `cli/preview.ts`?

Reply:
Done